plugin.php

```diff
@@ -2,7 +2,7 @@
 
 /*
   Plugin Name: Shop Standards
-  Version: 1.30.18
+  Version: 1.30.19
```

Comment:
I'm not sure this is a fix. Sounds more a new minor feature to me.

Reply:
Meh, not sure about that but to me it feels more like a fix to the already existing functionality.